be/src/vec/exec/vbroker_scanner.cpp

```diff
@@ -95,6 +91,10 @@ Status VBrokerScanner::_fill_dest_columns(const Slice& line,
         return Status::OK();
     }
 
+    if (!check_array_format(_split_values)) {
```

Comment:
I think it is not a good implementation. We can filter the invalid row in tablet_sink.cpp.

Reply:
As we disscuss, the 'tablet_sink.cpp' layer just can process the block which has parse the entire array. if the 'vbroker_scanner' layer can parse the array failed and return error, the 'tablet_sink.cpp' layer will not get the block to filter.

Reply:
You are right.